### ISSUE TITLE
forum.inc doesn't translate "Private Messages"

### ISSUE THREAD

**Issue by romw** [BOINC/boinc-ng#475]
**Reported by Ageless on 4 Aug 37886370 20:53 UTC**
In [14259]
```
echo "<p>Private messages: ", pm_notification($user); 
```
should be 
```
echo "<p>" . tra("Private messages") . ": ", pm_notification($user);
```
to show the "Private Messages" part in languages other than English.

**Comment by romw** [comment]
**Commented by Ageless on 8 Jun 37886386 21:46 UTC**
Thx Nick. :)

**Comment by romw** [comment]
**Commented by Rytis on 1 May 38181430 08:26 UTC**
(In [14861]) User web: translate some strings in forum.inc (fix #496).
